kn/fotos/urls.py

```diff
@@ -12,6 +13,8 @@
             views.fotoadmin_create_event, name='fotoadmin-create-event'),
         url(r'^fotos/api/?$',
             api.view, name='fotos-api'),
+        url(r'^fotos/view\.php$', views.compat_view),
+        url(r'^fotos/foto\.php$', views.compat_foto),
```

Comment:
`index.php` was ik vergeten, dank je! In mijn browsergeschiedenis wordt `index.php` inderdaad gebruikt, dus blijkbaar zijn er URLs die daar naar verwijzen.